src/libraries/System.Private.CoreLib/src/System/Byte.cs

```diff
@@ -392,7 +392,7 @@ object IConvertible.ToType(Type type, IFormatProvider? provider)
         static byte IDivisionOperators<byte, byte, byte>.operator /(byte left, byte right) => (byte)(left / right);
 
         // /// <inheritdoc cref="IDivisionOperators{TSelf, TOther, TResult}.op_CheckedDivision(TSelf, TOther)" />
-        // static byte IDivisionOperators<byte, byte, byte>.operator checked /(byte left, byte right) => checked((byte)(left / right));
+        static byte IDivisionOperators<byte, byte, byte>.operator checked /(byte left, byte right) => checked((byte)(left / right));
```

Comment:
What values of left and right would this throw an overflow exception for?

Reply:
For .NET/IL, Division for the primitives doesn't overflow and in fact there is no `div.ovf` instruction in IL. The behavior is that division by `zero` always throws a `DivideByZeroException` and that `MinValue / -1` always throws `OverflowException`.

For C#, the language formally defines `checked(x / y)` to throw `ArithmeticException` (or a subclass) for `MinValue / -1`. While it leaves `unchecked(x / y)` for the same to be "implementation-defined` on whether it throws or goes unreported and returns `x`.

Reply:
But byte can't be -1.  So is the checked on the rhs here doing anything?  Seems like you get identical behavior / asm with or without it, yes?

Reply:
Right. For byte (and the other primitives) this is a case we'd switch over to a DIM once that's available. .NET only exposes a single instruction for these and so there's no need for the primitives to actually do anything special.

We likely still want to support the concept on the interface for end users who may want to provide their own overflow behavior if that makes sense for their domain (although my opinion here isn't very strong for division in particular).

Reply:
> this is a case we'd switch over to a DIM once that's available

Cool.  Would it make sense then to remove `checked(...)` from this PR where it's not actually doing anything?  My reasoning is simply that it'll make it easier to spot all the operators we can remove once we have DIMs: if they're not using checked, remove them, if they are, keep them.  Up to you.

Reply:
Can do. I made sure to remove them from float/single/double, DateTime, and TimeSpan; so it makes sense to do the same for integer division

Reply:
Fixed.